indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -41,11 +32,23 @@
     {% endfor %}
 {% endmacro %}
 
-<div class="toolbar">
-    <div class="group right">
-        <a class="i-button js-reset-btn" type="button">
-            {%- trans %}Clear filters{% endtrans -%}
-        </a>
+<div class="action-box">
+    <div class="section">
+        <div class="icon icon-wrench"></div>
+        <div class="toolbar right">
+            <a class="i-button js-reset-btn icon-close warning" type="button">
+                {%- trans %}Clear filters{% endtrans -%}
+            </a>
+        </div>
+        <div class="text">
+            <div class="label">Customize registration list</div>
+            <div>
+                {% set filter_icon = '<i class="icon-filter"></i>' | safe %}
+                {%- trans filter_icon=filter_icon -%}
```

Comment:
I *think* you don't need to put the variable here if you really just use a plain variable inside the trans block.